.env.local

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_RECAPTCHA_SITE_KEY=""
+RECAPTCHA_SECRET_KEY=""
```

Comment:
Please rename it to be more meaningful.
```
GOOGLE_RECAPTCHA_SITE_KEY
GOOGLE_RECAPTCHA_SECRET_KEY
```

Reply:
Also don't forget the newline.

Reply:
For the name of `NEXT_PUBLIC`, I tend to make the token visible to the client. You can check about the exposing env variable [here](https://nextjs.org/docs/basic-features/environment-variables). Plus, the **reCaptcha** is specific enough because it's a product name of Google.

Reply:
Who's the client? We are the client though !? Then it's okay not to add the `google` prefix but better to remove the `NEXT_PUBLIC`. Try to name these variables by their sources.

Reply:
1. The `secret key` is definitely used in further development. Would it be nice to declare both secret and site key, wouldn't it?
2. According to the ref I provided, by default all environment variables loaded through .env.local are only available in the Node.js environment, meaning they won't be exposed to the browser. In order to expose a variable to the browser, it needs a `NEXT_PUBLIC` prefix and the site key is the public key. If you don't mind, please provide any information to support that we do not need this prefix. 
3. About the double quotes, it's provided because when you have the text like `ghjkligj89` so you can just copy and paste it inside the double quotes that already there. There's no different, have or don't have one.